docs/source/hooks/introduction.md

```diff
@@ -12,8 +12,8 @@ Hooks are a mechanism to add extra behaviour to Kedro's main execution in an eas
 
 A Hook consists of a Hook specification, and Hook implementation. To add Hooks to your project, you must:
 
-* Create or modify the file `<your_project>/src/<package_name>/hooks.py` to define a Hook implementation for an existing Kedro-defined Hook specification
-* Register your Hook implementation in the [`src/<your_project>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key
+* Create or modify the file `<project-name>/src/<package_name>/hooks.py` to define a Hook implementation for an existing Kedro-defined Hook specification
+* Register your Hook implementation in the [`src/<project-name>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key
```

Comment:
```suggestion
* Register your Hook implementation in the [`src/<package_name>/settings.py`](../kedro_project_setup/settings.md) file under the `HOOKS` key
```

Reply:
Changes made in [344a6fb](https://github.com/kedro-org/kedro/pull/1753/commits/344a6fb49d232371b8f592ee5ccc35d681bee043).